charmcraft/commands/store/store.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For further info, check https://github.com/canonical/charmcraft
+
+"""The Store API handling."""
+
+from collections import namedtuple
```

Comment:
could you use `typing.NamedTuple` instead?
The 3.5-compatible version is very similar to your usage here (3.6+ is even more interesting but alas)

Reply:
I ended up writing #70 for this and related